pyoptsparse/pyOpt_optimizer.py

```diff
@@ -758,28 +757,14 @@ def _assembleObjective(self):
 
         return np.real(np.squeeze(ff))
 
-    def _createSolution(self, optTime, sol_inform, obj, xopt, multipliers=None):
+    def _createSolution(self, optTime, sol_inform, obj, xopt, multipliers=None) -> Solution:
```

Comment:
It looks like there is some significant refactoring here that is not just type hints. Maybe we should update the PR title/description?

Reply:
I forgot about this.. I defined these class attributes explicitly in the Solution object, before we were just injecting these in random places which was pretty poor. I will update the descriptions.